axelrod/__init__.py

```diff
@@ -1,4 +1,7 @@
 from __future__ import absolute_import
+import sys
+
+on_windows = sys.platform.startswith("win")
```

Comment:
Not an educated reason: would `os.name` be better? I have to check for `nt` or something like that right?